test/unit/Schema.js

```diff
@@ -961,7 +961,6 @@ describe("Schema", () => {
 			expect(result.map((item) => ({"name": item.name, "dynamodbType": item.dynamodbType}))).to.eql([{"name": "String", "dynamodbType": "S"}, {"name": "Buffer", "dynamodbType": "B"}]);
 		});
 	});
-
```

Comment:
Let's revert this change.

Reply:
Resolved in https://github.com/dynamoose/dynamoose/pull/1079/commits/60628df9e7ec5749d723d8b7165a4da7b259934e